src/components/Table/GridTable.tsx

```diff
@@ -1093,6 +1099,10 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
     // Maybe add the sticky header styles
     ...((isHeader || isTotals) && stickyHeader ? Css.sticky.topPx(stickyOffset).z2.$ : undefined),
     ...getNestedCardStyles(row, openCardStyles, style, isActive),
+    ...{
+      "> .revealOnRowHover > *": Css.invisible.$,
+      ":hover > .revealOnRowHover > *": Css.visible.$,
```

Comment:
**suggestion:** Can these use the `revealOnRowHoverClass` const?
```
` > .${revealOnRowHoverClass} > *`: Css.invisible.$,
...
```

Reply:
ah I tried using that initially but kept getting a type error. Seems like it doesn't like using a template string as key?

Reply:
Maybe try:
```
[` > .${revealOnRowHoverClass} > *`]: Css.invisible.$,
...
```

Reply:
ah that works, interesting that it needs to be wrapped